pkg/component/logging/vali/templates/vali-config.yaml.tpl

```diff
@@ -1,4 +1,3 @@
-auth_enabled: {{ .AuthEnabled }}
```

Comment:
This YAML file is no longer a template. Rename it to `pkg/component/logging/vali/templates/vali-config.yaml` and drop the templating in https://github.com/gardener/gardener/blob/d4a4cbada3cd9b18a3e9ea83d566059b741933ab/pkg/component/logging/vali/vali.go#L574-L577

Reply:
feedback is applied